sample/sample8-middlewares/UserController.ts

```diff
@@ -0,0 +1,56 @@
+import {Request} from "express";
+import {Controller} from "../../src/decorator/controllers";
+import {Get, Post, Put, Patch, Delete, Use} from "../../src/decorator/methods";
+import {Req, Next} from "../../src/decorator/params";
+
+@Controller()
+export class UserController {
+
+
+    @Use()
+    middleware( @Next() next: any) {
```

Comment:
as I understood this method `middleware` will be called before each route in this specific controller, am I right?